metricbeat/docs/modules/windows.asciidoc

```diff
@@ -20,13 +21,8 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: windows
-  metricsets: ["perfmon"]
   period: 10s
   perfmon.counters:
```

Comment:
`perfmon` requires configuration so maybe it shouldn't be a "default". You must configure `perfmon.counters` in order for it to be useful so I think this example needs improvement.

The `service` metricset does not require any configuration so it might be a good default. But TBH the default period of 10s is a excessive. I'd probably increase it 60s at a minimum or drop events for healthy services.

Reply:
+1 on the proposal. As we have now also a `config.reference.yml` we can set the period to 60s (non default) in the config.yml. Also we can add a processor to drop the healthy events with a processor in the `config.yml`.

Reply:
Agree with the proposals, thanks @andrewkroh 

Would it be ok to set a different default for the period in the initialization of this module?

Reply:
It would be possible but as the default period is possible, I would not do that and use the `config.yml` for it. This keeps the code and defaults simple, but still allows us to overwrite them.

Reply:
Oh, period is module-wide :thinking: it shouldn't be set from the metricset.

Reply:
You can define a module multiple times in the config if you want to have different periods for different metricsets.